code/game/gamemodes/cult/hell_universe.dm

```diff
@@ -97,20 +98,13 @@ In short:
 
 		A.updateicon()
 
-/datum/universal_state/hell/proc/OverlaySet()
-/*
-	for(var/turf/space/spess in world)
-		spess.overlays += "hell01"
-	*/
-	var/image/I = image("icon" = 'icons/turf/space.dmi', "icon_state" = "hell01", "layer" = 10)
-	var/area/space = locate(/area)
-	if(space.name == "Space")
-		space.overlays += I
-
-/datum/universal_state/hell/proc/AmbientSet()
+/datum/universal_state/hell/proc/OverlayAndAmbientSet()
```

Comment:
Shouldn't you move this proc to universal_state, considering it's defined twice?